code/modules/maps/overmap/planets/_virgo3b.dm

```diff
@@ -1,15 +1,14 @@
 
 /datum/atmosphere/planet/virgo3b
-	base_gases = list(
-	/datum/gas/nitrogen = 0.16,
-	/datum/gas/phoron = 0.72,
-	/datum/gas/carbon_dioxide = 0.12
+	base = list(
+		/datum/gas/nitrogen = 0.16,
+		/datum/gas/phoron = 0.72,
```

Comment:
why are gases here not switched to IDs?

Reply:
compile time checking
paths are just Better if we can spare the performance
i don't do paths on objects if they need to be resolved most of the time because it doesn't check; when we go to 515 we can do /path::id but for now it's just path and it auto resolves on New()